src/pages/workspaces/workspace/Data.js

```diff
@@ -477,7 +477,9 @@ const WorkspaceData = _.flow(
     try {
       setSnapshotDetails(_.set([snapshotName, 'error'], false))
       const entities = await Ajax(signal).Workspaces.workspace(namespace, name).snapshotEntityMetadata(googleProject, snapshotName)
-      setSnapshotDetails(_.set([snapshotName, 'entityMetadata'], entities))
+      //Prevent duplicate id columns
+      const entitiesWithoutIds = _.mapValues(value => _.update(['attributeNames'], _.without([value.idName]), value), entities)
```

Comment:
```suggestion
      const entitiesWithoutIds = _.mapValues(entity => _.update(['attributeNames'], _.without([entity.idName]), value), entities)
```

small readability nit! could we name the "value" as "entity"?